vavr/src/main/java/io/vavr/control/Validation.java

```diff
@@ -468,11 +477,11 @@ default void forEach(Consumer<? super T> action) {
      * @return an instance of type U
      * @throws NullPointerException if fInvalid or fValid is null
      */
-    default <U> U fold(Function<? super E, ? extends U> fInvalid, Function<? super T, ? extends U> fValid) {
+    default <U> U fold(Function<Seq<? extends E>, ? extends U> fInvalid, Function<? super T, ? extends U> fValid) {
         Objects.requireNonNull(fInvalid, "fInvalid is null");
         Objects.requireNonNull(fValid, "fValid is null");
         if (isInvalid()) {
-            final E error = this.getError();
+            final Seq<? extends E> error = this.getErrors();
```

Comment:
will change to the following after you change the return type of `getErrors()` as stated above:

```java
final Seq<E> error = this.getErrors();
```